### DISCUSSION TITLE
Not able to access benefit status page

### DISCUSSION THREAD

**Discussion by EncVar** [community/community#148021] [GitHub Education]
### Hi everyone,

On https://education.github.com/pack appears `We're still processing your application. Once this is complete you will have access to the student developer pack offers` and it has been stayed at this status for a couple of months. Also, it keeps redirecting to the pricing page automatically when accessing benefit page.

**Comment by CyberSphinxxx** [comment]
Hi @EncVar,

Your application may still be in the queue for processing, which can sometimes take a while. However, if it has been a couple of months, it seems your application might be stuck. It's a good idea to contact GitHub Education Support directly [here](https://support.github.com/contact/education) for clarification or assistance.  

In the meantime, ensure that all required documents and information in your application are complete and accurate.
